module2/lessons/intro_to_mvc.md

```diff
@@ -99,30 +96,30 @@ We saw earlier that the Model is responsible for interaction with the database.
+Serializers are typically given a resource or collection of resources from our database to iterate over. It's appropriate for the Serializer to call instance methods from the Model class if needed. A Serializer should not call *class* methods except in extremely rare cases where a data builder requires it, such as working with associated database resources (you haven't learned about these yet, so it's okay if you aren't sure what that means).
 
 <section class="call-to-action">
-### MVC and Forms
+### MVC and data
 
-In MVC applications, we can get and manipulate data on an html page in multiple ways. You've collected data from a user with JavaScript eventListeners as part of the Hang In There project, and now we are using erb to get data from our view to our controller, as well as from our controller to our view.
+In MVC applications, we can get and manipulate data in multiple ways. Sometimes we use HTML and forms. You've collected data from a user with JavaScript eventListeners as part of the Hang In There project, and now we are using JSON from HTTP requests to send data to our controller, as well as from our controller to our view layer, which in our case is JSON.
 </section>
 
 ### Controllers -- Business Logic, or Application Logic
 
-Our Controllers are the "traffic cop" between our Models and our Views. Based on the incoming request, each controller method knows precisely which Model(s) it needs to utilize to fetch or write data, and will generally hand that data off to a View for presentation.
+Our Controllers are the "traffic cop" between our Models and our View layer. Based on the incoming request, each controller method knows precisely which Model(s) it needs to utilize to fetch or write data, and will generally hand that data off to a Serializer for presentation.
 
-Controllers should limit their database actions to very simple lookups, or creation of a resource. A controller should not do very much data manipulation, that "data logic" is the role of the Model. Likewise, the controller should pre-fetch as much data as possible so the View does not call Class methods from the Models.
+Controllers should limit their database actions to very simple lookups, or creation of a resource. A controller should not do very much data manipulation, that "data logic" is the role of the Model. Likewise, the controller should pre-fetch as much data as possible so the Serializer or HTML template does not call Class methods from the Models.
 
 ### Experiment
 
-- Add a new route and view so that the user can successfully visit `localhost:3000/easteregg` and see a new view of your choice.
+- Add a new route and serializer so that the user can successfully visit `localhost:3000/easteregg` and see a new JSON formatting of your choice.
```

Comment:
Noticing we never stated it in the previous lesson, but should we include a controller action here to in order to hammer home that to expose the endpoint, we need these 3 things?
`Add a new route, controller action, and a serializer so that . . . `

Reply:
I might just make this update if no one objects and then I can merge it in